__std__/nix/plutus/packages/default.nix

```diff
@@ -1,4 +1,6 @@
-{ inputs, cell }@organelle:
+{ inputs, cell }@block:
 {
-  all-components-with-haddock = import ./all-components-with-haddock.nix organelle;
+  all-components-with-haddock = import ./all-components-with-haddock.nix block;
+
+  ghc = import ./ghc.nix block;
```

Comment:
unsure we even want this as a standalone packge?

Reply:
obsoleted by latest commit

Reply:
still seems to be here